src/Stripe.net/Entities/InvoiceLineItems/InvoiceLineItem.cs

```diff
@@ -58,12 +48,6 @@ public List<Discount> Discounts
         internal List<ExpandableField<Discount>> InternalDiscounts { get; set; }
         #endregion
 
-        /// <summary>
-        /// The amount of discount calculated per discount for this line item.
-        /// </summary>
-        [JsonProperty("discount_amounts")]
-        public List<CreditNoteLineItemDiscountAmount> DiscountAmounts { get; set; }
-
```

Comment:
Why is this gone? it's in the reference: https://stripe.com/docs/api/invoices/line_item#invoice_line_item_object-discount_amounts

Reply:
It was moved up to be in order and renamed, added to pr comment.

Reply:
ah sorry my bad, I didn't notice they moved since the other had no comment. All good, I got worried for a minute